src/app/components/add-task-form/add-task-form.component.ts

```diff
@@ -14,36 +15,43 @@ import { Router, RouterModule } from '@angular/router';
   templateUrl: './add-task-form.component.html',
   styleUrl: './add-task-form.component.scss'
 })
-export class AddTaskFormComponent implements OnInit {
+export class AddTaskFormComponent implements OnInit, OnDestroy {
   typeOptions: string[] = [
     "Story",
     "Task"
   ]
   formGroup!: FormGroup;
```

Comment:
non null assertion

Reply:
instead of adding non null, you can directly assign the formbuilder logic here instead of ngOnInit